handlers/handlers_test.go

```diff
@@ -239,7 +255,7 @@ func TestUnitReadFailure(t *testing.T) {
 		So(err, ShouldBeNil)
 
 		Convey("When read is called", func() {
-			read(w, req, cfg, mockedZebedeeClient, mockedRendererClient, mockedSearchClient, accessToken, collectionID, englishLang, *mockCacheList, "search")
+			read(w, req, cfg, mockedZebedeeClient, mockedRendererClient, mockedSearchClient, accessToken, collectionID, englishLang, *mockCacheList, "search", false)
```

Comment:
Not for here (maybe) but I think these function signatures for all the handlers are getting quite big and are all quite similar. Do you think they should be a method with a receiver type?

Reply:
Take the point but later :)